<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>398</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>398</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>398</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="styleSheet">
   <string>background-color: rgb(58,58,58); 
selection-background-color: rgb(67, 67, 67);
color: rgb(255, 255, 255);
</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>340</y>
     <width>411</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string>background-image: url(:/Resources/SeparatorBottom.png); 
border: none;</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>358</y>
     <width>411</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string>background-color: rgb(46, 46, 46);
border:none;</string>
   </property>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>236</x>
      <y>5</y>
      <width>75</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string>background-image: url(:/Resources/ButtonNormal.png); 
border: none;
color: rgb(255, 255, 255);
font: bold 10pt &quot;Lucida Grande&quot;;</string>
    </property>
    <property name="text">
     <string>Apply</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>322</x>
      <y>5</y>
      <width>75</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string>background-image: url(:/Resources/ButtonNormal.png); 
border: none;
color: rgb(255, 255, 255);
font: bold 10pt &quot;Lucida Grande&quot;;</string>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
  </widget>
  <widget class="QTabWidget" name="tabs_">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>321</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTabWidget::pane {
	border: 1px solid gray;
	border-radius: 4px; 
	padding: 4px;
}
QTabWidget::tab-bar {
left: 5px; 
}
QTabBar::tab {
background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #8A8A8A, stop: 0.5 #3A3A3A, stop: 1.0 #393939);
border: 1px solid #C4C4C3;
border-bottom: none;
border-top-left-radius: 4px;
border-top-right-radius: 4px;
min-width: 60px;
height: 15px;
padding-left: 10px;
padding-top: 2px;
padding-right: 10px;
padding-bottom: 2px;
}
QTabBar::tab:hover {
background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #9A9A9A, stop: 0.5 #5A5A5A, stop: 1.0 #595959);
}
QTabBar::tab:selected {
background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #CACACA, stop: 0.5 #8A8A8A, stop: 1.0 #898989);
border-color: #9B9B9B;
border-bottom-color: #C2C7CB; 
}
QTabBar::tab:!selected {
}
QTabBar::tab:selected {
margin-left: -4px;
margin-right: -4px;
}
QTabBar::tab:first:selected {
margin-left: 0; 
}
QTabBar::tab:last:selected {
margin-right: 0;
}
QTabBar::tab:only-one {
margin: 0; 
}</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Screen</string>
    </attribute>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>80</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lucida Grande</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>up to 20 fps</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>311</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lucida Grande</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>This option allows you to display part of your screen on all connected devices.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lucida Grande</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Frames per second:</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="screenshotsMode_">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>141</width>
       <height>19</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lucida Grande</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string> Spread screenshots</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>143</y>
       <width>330</width>
       <height>2</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string>background-image: url(:/Resources/SeparatorMiddle.png); 
border: none;</string>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="screenshotsFPS_">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>80</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lucida Grande</family>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="sfbComprLabel_">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>110</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lucida Grande</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>100%</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_44">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lucida Grande</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Compression level:</string>
     </property>
    </widget>
    <widget class="QSlider" name="sfbCompr_">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>110</y>
       <width>161</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="service_">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>260</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Helvetica</family>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>160</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lucida Grande</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Medium</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>230</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lucida Grande</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Traffic encryption:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>341</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lucida Grande</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Authentication feature is enabled and allows a device to get control over host machine based on predefined rule.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>263</y>
       <width>46</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lucida Grande</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Port:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>263</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lucida Grande</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>[1024 .. 65535]</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lucida Grande</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Security:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>230</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lucida Grande</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Disabled</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>screenshotsMode_</tabstop>
  <tabstop>screenshotsFPS_</tabstop>
  <tabstop>pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>apply()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>247</x>
     <y>298</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>200</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>377</x>
     <y>313</y>
    </hint>
    <hint type="destinationlabel">
     <x>387</x>
     <y>195</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>apply()</slot>
  <slot>compressionChanged(int)</slot>
 </slots>
</ui>
